kiva/_cython_speedups.pyx

```diff
@@ -67,7 +67,7 @@ def points_in_polygon(pts, poly_pts, use_winding=False):
 
     cdef double[:, ::1] pts_view = pts
     cdef double[:, ::1] poly_pts_view = poly_pts
-    cdef uint8_t[::1] results = np.zeros(len(pts), dtype=np.uint8)
+    cdef np.uint8_t[::1] results = np.zeros(len(pts), dtype=np.uint8)
```

Comment:
Are you sure that shouldn't be `np.npy_uint8`?

Reply:
I think that magically transmogrifies to `npy_uint8` somewhere deep under the Cython hood...

Reply:
https://github.com/cython/cython/blob/master/Cython/Includes/numpy/__init__.pxd#L750

Reply:
Indeed. Thanks for clearing that up